index.html

```diff
@@ -164,11 +164,11 @@ <h2>Rent Strikes</h2>
           <p><strong>Start: </strong>{{Start}}</p>
         </div>
         {{/Start}}
-        {{#Landlord}}
+        {{#Resources}}
```

Comment:
@erinmcel I don't see a "Resources" field in the rent strikes spreadsheet. This change assumes there is one. Is there something I'm missing?

Reply:
Just changed it now.